host/libhackrf/src/hackrf.h

```diff
@@ -182,88 +182,170 @@ typedef struct {
 } hackrf_m0_state;
 
 struct hackrf_device_list {
-	char **serial_numbers;
-	enum hackrf_usb_board_id *usb_board_ids;
-	int *usb_device_index;
+	char** serial_numbers;
+	enum hackrf_usb_board_id* usb_board_ids;
+	int* usb_device_index;
 	int devicecount;
-	
-	void **usb_devices;
+
+	void** usb_devices;
 	int usb_devicecount;
 };
+
 typedef struct hackrf_device_list hackrf_device_list_t;
 
 typedef int (*hackrf_sample_block_cb_fn)(hackrf_transfer* transfer);
 
 #ifdef __cplusplus
-extern "C"
-{
+extern "C" {
 #endif
 
 extern ADDAPI int ADDCALL hackrf_init();
+
 extern ADDAPI int ADDCALL hackrf_exit();
 
 extern ADDAPI const char* ADDCALL hackrf_library_version();
+
 extern ADDAPI const char* ADDCALL hackrf_library_release();
 
 extern ADDAPI hackrf_device_list_t* ADDCALL hackrf_device_list();
-extern ADDAPI int ADDCALL hackrf_device_list_open(hackrf_device_list_t *list, int idx, hackrf_device** device);
-extern ADDAPI void ADDCALL hackrf_device_list_free(hackrf_device_list_t *list);
- 
+
+extern ADDAPI int ADDCALL hackrf_device_list_open(
+	hackrf_device_list_t* list,
+	int idx,
+	hackrf_device** device);
+
+extern ADDAPI void ADDCALL hackrf_device_list_free(hackrf_device_list_t* list);
+
 extern ADDAPI int ADDCALL hackrf_open(hackrf_device** device);
-extern ADDAPI int ADDCALL hackrf_open_by_serial(const char* const desired_serial_number, hackrf_device** device);
+
+extern ADDAPI int ADDCALL hackrf_open_by_serial(
+	const char* const desired_serial_number,
+	hackrf_device** device);
 extern ADDAPI int ADDCALL hackrf_close(hackrf_device* device);
- 
-extern ADDAPI int ADDCALL hackrf_start_rx(hackrf_device* device, hackrf_sample_block_cb_fn callback, void* rx_ctx);
+
+extern ADDAPI int ADDCALL hackrf_start_rx(
+	hackrf_device* device,
+	hackrf_sample_block_cb_fn callback,
+	void* rx_ctx);
+
 extern ADDAPI int ADDCALL hackrf_stop_rx(hackrf_device* device);
- 
-extern ADDAPI int ADDCALL hackrf_start_tx(hackrf_device* device, hackrf_sample_block_cb_fn callback, void* tx_ctx);
+
+extern ADDAPI int ADDCALL hackrf_start_tx(
+	hackrf_device* device,
+	hackrf_sample_block_cb_fn callback,
+	void* tx_ctx);
+
 extern ADDAPI int ADDCALL hackrf_stop_tx(hackrf_device* device);
 
-extern ADDAPI int ADDCALL hackrf_get_m0_state(hackrf_device* device, hackrf_m0_state* value);
-extern ADDAPI int ADDCALL hackrf_set_tx_underrun_limit(hackrf_device* device, uint32_t value);
-extern ADDAPI int ADDCALL hackrf_set_rx_overrun_limit(hackrf_device* device, uint32_t value);
+extern ADDAPI int ADDCALL hackrf_get_m0_state(
+	hackrf_device* device,
+	hackrf_m0_state* value);
+
+extern ADDAPI int ADDCALL hackrf_set_tx_underrun_limit(
+	hackrf_device* device,
+	uint32_t value);
+
+extern ADDAPI int ADDCALL hackrf_set_rx_overrun_limit(
+	hackrf_device* device,
+	uint32_t value);
```

Comment:
The `SeparateDefinitionBlocks` option doesn't add spacing between function declarations. Since this file is directly user-facing, I've manually added empty lines between the declarations to make it easier to read. Having them bunched up next to each other works well when they're each on single lines, but less so when they're wrapped.

In any case these should all be gaining doxygen comments at some point.